modules/examples/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentityExamples.java

```diff
@@ -408,6 +441,342 @@ public static void main(String[] args) throws Exception {
+                        .build();
+
+            Response<TrustedProfilesList> response = service.listProfile(listProfileOptions).execute();
+            TrustedProfilesList profiles = response.getResult();
+
+            System.out.println(profiles);
+
+            // end-get_list_profile
+
+        } catch (ServiceResponseException e) {
+            logger.error(String.format("Service returned status code %s: %s\nError details: %s",
+                    e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
+        }
+
+        try {
+            System.out.println("updateProfile() result:");
+
+            // begin-update_profile
+
+            String newDescription = "updated description";
+            UpdateProfileOptions updateProfileOptions = new UpdateProfileOptions.Builder()
+                    .profileId(profileId)
+                    .ifMatch(profileEtag)
+                    .description(newDescription)
+                    .build();
+
+            Response<TrustedProfile> response = service.updateProfile(updateProfileOptions).execute();
+            TrustedProfile profile = response.getResult();
+
+            System.out.println(profile);
+
+            // end-update_profile
+
+        } catch (ServiceResponseException e) {
+            logger.error(String.format("Service returned status code %s: %s\nError details: %s",
+                    e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
+        }
+
+        try {
+            System.out.println("createClaimRule() result:");
+
+            // begin-create_claimrule
+
+            ProfileClaimRuleConditions condition = new ProfileClaimRuleConditions.Builder()
+                    .claim("blueGroups")
+                    .operator("EQUALS")
+                    .value("\"cloud-docs-dev\"")
+                    .build();
+
+            List<ProfileClaimRuleConditions> conditions = new ArrayList<>();
+            conditions.add(condition);
+
+            CreateClaimRuleOptions createClaimRuleOptions = new CreateClaimRuleOptions.Builder()
+                    .profileId(profileId)
+                    .type(claimRuleType)
+                    .realmName(realmName)
+                    .expiration(43200)
+                    .conditions(conditions)
+                    .build();
+
+            Response<ProfileClaimRule> response = service.createClaimRule(createClaimRuleOptions).execute();
+            ProfileClaimRule claimRule = response.getResult();
+            claimRuleId = claimRule.getId();
+
+            System.out.println(claimRule);
+
+            // end-create_claimrule
+
+        } catch (ServiceResponseException e) {
+            logger.error(String.format("Service returned status code %s: %s\nError details: %s",
+                    e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
+        }
+
+        try {
+            System.out.println("getClaimRule() result:");
+
+            // begin-get_claimrule
```

Comment:
```suggestion
            // begin-get_claimRule
```

Reply:
And other occasions like this. These tags have to be consistent across SDKs. Sorry for being picky, but it can break the public facing reference.